apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/__snapshots__/index-table.spec.tsx.snap

```diff
@@ -2826,6 +2917,19 @@ exports[`IndexTable when the table setup is valid when all properies are include
               <div
                 class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-3tbyo6-MuiGrid-root"
               >
+                <div
```

Comment:
I'll have to take a look at this. Note that Justin has a branch working on pagination. Hopefully this doesn't interfere with that.

Reply:
This one is just an idea, it can go on a small branch.  The main idea is that when you filter, you're able to see the new total as well as the entire total for the resource.  Good for knowing how many terms a certain contributor is connected to, and other things like that.

Reply:
Yea absolutely a good idea. Just want to make sure this is on its own PR and there aren't merge conflicts with Justin's similar work. Let's make a story for this and put our heads together on it.

Reply:
Also note that I anticipate moving to an "active search" pattern or else supporting pagination in Redux in the near future.